scripts/valgrind.supp

```diff
@@ -15,6 +15,14 @@
    fun:posix_new_thread
    fun:arch_new_thread
 }
+{
+   native_sim CPU thread emulation no thread cleanup
+   Memcheck:Leak
+   match-leak-kinds: reachable,possible
+   ...
+   fun:nct_new_thread
```

Comment:
Weird, how did you build to get this not suppressed?
The call stack should be:
```
==1641255==    by 0x804D0B2: nct_new_thread (nct.c:413)
==1641255==    by 0x804B2B6: posix_new_thread (posix_core_nsi.c:48)
==1641255==    by 0x804B190: arch_new_thread (thread.c:55)
```
so it should match the rule above.
(maybe some optimization bypassed the call to posix_new_thread)

Reply:
Anyhow, I guess we can do 
```
diff --git a/scripts/valgrind.supp b/scripts/valgrind.supp
index 330ec51fefd..46703150052 100644
--- a/scripts/valgrind.supp
+++ b/scripts/valgrind.supp
@@ -12,7 +12,8 @@
    Memcheck:Leak
    match-leak-kinds: reachable,possible
    ...
-   fun:posix_new_thread
+   fun:nct_new_thread
+   ...
    fun:arch_new_thread
 }
``` 
which should match either